circle.yml

```diff
@@ -39,11 +39,8 @@ machine:
 
 checkout:
   post:
-    - |
-      git clone --depth 1 ${ST2_PACKAGES_REPO} /home/ubuntu/st2/st2-packages
-      cd /home/ubuntu/st2/st2-packages
-      git checkout ${CIRCLE_BRANCH} || true
```

Comment:
Actually not having the remote branch is a failure. It should fail loud and clear rather than falling back to master.

So yeah some build might fail due to a race but its easy to get out of the race.